notebooks/Add GPS Metadata Fields.ipynb

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
<p>I would have a global variable at the top of the script like:</p><p><code>FEATURE_CLASS_PATH = r"C:\..."</code></p><p>Then the user can just change that thing at the top and run it</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/field-maps-scripts/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>